source/migration.txt

```diff
@@ -0,0 +1,176 @@
+    network bandwidth. Provision the migration host(s) with the following minimum configuration:
+
+    .. list-table::
+       :header-rows: 1
+       :widths: 20 40
+
+       * - Number of VMs
+         - Total of 3 VMs, 2 for sharded and 1 for RS
+
+       * - Purpose
+         - Mongosync
+
+       * - Location
+         - Must access on-prem and public cloud
+
+       * - CPU
+         - 8
+
+       * - Memory
+         - 32 GB
+
+       * - OS
+         - OS - 64-bit
+
+       * - Disk Size
+         - Disk enough for logging
+
+    To ensure a smooth migration process, confirm that the source cluster's oplog size is adequate to 
+    cover the entire migraiton duration.
+
+  For full migration recommendations and instructions, see :ref:`c2c-pull-live-migration`.
+
+* **Push data into Atlas.** |com| push data to |service| using a secure :term:`link-token` 
+  without requiring access to the source {+cluster+} through the {+cluster+}'s firewall.
+  During migration, |service| continuously syncs real-time data between the source 
+  and destination {+clusters+} until cutover. The following
+  considerations apply:
+  
+  - Data is synchronized in one direction only: changes made to the destination won't 
+    reflect back on the source.
+  - Supports :ref:`VPC peering <vpc-peering>` and :ref:`private endpoints <private-endpoint>`.
+  - Source and destination {+cluster+} topologies must match. For example, both 
+    must be replica sets or sharded clusters with the same number of shards.
+  - Ensure that you connect to your |service| {+cluster+} from all client hardware where your 
+    applications run. Testing your connection string helps ensure that your data migration 
+    process completes with minimal downtime.
+  
+  For full migration recommendations and instructions, see :ref:`c2c-push-live-migration`.
+
+Monitoring Migrations
+---------------------
+
+.. include:: /includes/cloud-docs/shared-migration-monitoring-description.rst
+
+To learn more, see :ref:`monitor-migrations`.
+
+.. _arch-center-manual-migration:
+
+Manual Migration Methods
+------------------------
+
+If |service| live migration can't satisfy the constraints of your migration requirements, 
+you can bring data from existing MongoDB deployments, ``JSON``, or ``CSV`` files
+into |service| using one of the following tools that you run outside of |service|.
+
+.. include:: /includes/cloud-docs/shared-migration-tools-table.rst
+
+If you are required to use either |service| VNet peering or Private Link configurations, and/or 
+don't want to allow direct connection from a third party to its source cluster, or if you 
+don't already have or don't want to import the source cluster in |onprem| or |mms|, then MongoDB 
+recommends the mongosync approach.
+
+If you have relatively small datasets (<300 GB) to migrate, and can afford application downtime 
+for an extended time period, then MongoDB recommends the mongodump and mongorestore approach.
```

Comment:
```suggestion
for an extended time period, then MongoDB recommends the ``mongodump`` and ``mongorestore`` approach.
```
is there anywhere in Atlas docs or server manual where we can link to cover this approach?